docs/advanced-guide/injecting-databases-drivers/page.md

```diff
@@ -712,4 +712,118 @@ func queryDataPoints(c *gofr.Context) (any, error) {
 	}
 	return queryResp.QueryRespCnts, nil
 }
+```
+ScyllaDB
+
+
+GoFr supports injecting ScyllaDB to facilitate interaction with ScyllaDB REST APIs.Implementations adhering to the ScyllaDB interface can be registered with app.AddScyllaDB(),enabling applications to leverage ScyllaDB for time-series data management through gofr.Context
+```go
+type ScyllaDB interface {
+// Query executes a CQL (Cassandra Query Language) query on the ScyllaDB cluster
+// and stores the result in the provided destination variable `dest`.
+// Accepts pointer to struct or slice as dest parameter for single and multiple 
+Query(dest any, stmt string, values ...any) error
+// QueryWithCtx executes the query with a context and binds the result into dest parameter.
+// Accepts pointer to struct or slice as dest parameter for single and multiple rows retrieval respectively.
+QueryWithCtx(ctx context.Context, dest any, stmt string, values ...any) error
+// Exec executes a CQL statement (e.g., INSERT, UPDATE, DELETE) on the ScyllaDB cluster without returning any result.
+Exec(stmt string, values ...any) error
+// ExecWithCtx executes a CQL statement with the provided context and without returning any result.
+ExecWithCtx(ctx context.Context, stmt string, values ...any) error
+// ExecCAS executes a lightweight transaction (i.e. an UPDATE or INSERT statement containing an IF clause).
+//If the transaction fails because the existing values did not match, the previous values will be stored in dest.
+//Returns true if the query is applied otherwise false.
+// Returns false and error if any error occur while executing the query.
+// Accepts only pointer to struct and built-in types as the dest parameter.
+ExecCAS(dest any, stmt string, values ...any) (bool, error)
+// NewBatch initializes a new batch operation with the specified name and batch type.
+NewBatch(name string, batchType int) error
+// NewBatchWithCtx takes context,name and batchtype and return error.
+NewBatchWithCtx(_ context.Context, name string, batchType int) error
+// BatchQuery executes a batch query in the ScyllaDB cluster with the specified name, statement, and values.
+BatchQuery(name, stmt string, values ...any) error
+// BatchQueryWithCtx executes a batch query with the provided context.
+BatchQueryWithCtx(ctx context.Context, name, stmt string, values ...any) error
+// ExecuteBatchWithCtx executes a batch with context and name returns error.
+ExecuteBatchWithCtx(ctx context.Context, name string) error
+// HealthChecker defines the HealthChecker interface.
+HealthChecker
+}
```

Comment:
Please format this piece of code. You can put it your IDE, format, then paste it back here